### PR TITLE
feature prototype Login-screen

### PR DESCRIPTION
**Pull request by at-duytran**
login screen

### CODE REVIEW

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+        // Do any additional setup after loading the view.
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+
```
Comment (at-daiho) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: WARNING - Limit vertical whitespace to a single empty line. Currently 2.

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+
```
Comment (at-daiho) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: WARNING - Lines should not have trailing whitespace.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 2
Serious: 0
Warnings: 2

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/Model/MyModel.swift [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+//  MyModel.swift
+//  MyApp
+//
+//  Created by Asiantech on 7/31/17.
+//  Copyright © 2017 Asian Tech Co., Ltd. All rights reserved.
+//
+
+import Foundation
+
+class MyModel {
```
Comment (at-truongnguyen) [inline MyApp/Model/MyModel.swift]: @at-duytran What is this?

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+final class LoginViewController: UIViewController {
+
+    @IBOutlet private weak var userNameTextField: UITextField!
+    @IBOutlet private weak var passWordTextField: UITextField!
+    @IBOutlet private weak var loginButton: UIButton!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.navigationController?.isNavigationBarHidden = true
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: remove `self` please

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return .lightContent
+    }
+
+    @IBAction private func LoginButtonTouchUpInside(_ sender: Any) {
+        validationAccount()
+    }
+
+    private func validationAccount() {
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: why `validate` data in `View`?

File: MyApp/View/Controllers/Login/LoginViewController.swift [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+        // Custom textfield placeholder
+        if let usernamePlaceHolder = userNameTextField.placeholder {
+            userNameTextField.attributedPlaceholder = NSAttributedString(string: usernamePlaceHolder, attributes: [NSForegroundColorAttributeName: UIColor.white])
+        }
+        if let passwordPlaceHolder = passWordTextField.placeholder {
+            passWordTextField.attributedPlaceholder = NSAttributedString(string: passwordPlaceHolder, attributes: [NSForegroundColorAttributeName: UIColor.white])
+        }
+    }
+
+    private func customButton() {
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Login/LoginViewController.swift]: Should name it to `setupUI`

**Pull request merged**